cpp/cmake/thirdparty/get_libcugraphops.cmake

```diff
@@ -14,26 +14,50 @@
 # limitations under the License.
 #=============================================================================
 
-function(find_and_configure_cugraphops)
+set(CUGRAPH_MIN_VERSION_cugraph_ops "${CUGRAPH_VERSION_MAJOR}.${CUGRAPH_VERSION_MINOR}.00")
+set(CUGRAPH_BRANCH_VERSION_cugraph_ops "${CUGRAPH_VERSION_MAJOR}.${CUGRAPH_VERSION_MINOR}")
 
-    if(TARGET cugraphops::cugraphops)
-        return()
-    endif()
+function(find_and_configure_cugraph_ops)
 
-    rapids_find_generate_module(cugraphops
-        HEADER_NAMES            graph/sampling.hpp
-        LIBRARY_NAMES           cugraph-ops++
-        INCLUDE_SUFFIXES        cugraph-ops
-        BUILD_EXPORT_SET    cugraph-exports
-        INSTALL_EXPORT_SET  cugraph-exports
-    )
+    set(oneValueArgs VERSION FORK PINNED_TAG CLONE_ON_PIN)
+    cmake_parse_arguments(PKG "" "${oneValueArgs}" "" ${ARGN} )
+
+    if(PKG_CLONE_ON_PIN AND NOT PKG_PINNED_TAG STREQUAL "branch-${CUGRAPH_BRANCH_VERSION_cugraph_ops}")
+        message("Pinned tag found: ${PKG_PINNED_TAG}. Cloning cugraph-ops locally.")
+        set(CPM_DOWNLOAD_cugraph-ops ON)
+    endif()
 
-    rapids_find_package(cugraphops
```

Comment:
We should keep this component so we can find versions of cugraph-ops that don't provide the cmake `cugraph-ops-config.cmake` file. Once that has been merged we can come back and only use `rapids_cpm_find`.